modules/kmm-hub-installing-kmm-hub-olm.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-hub-installing-kmm-hub-olm_{context}"]
+= Installing KMM-Hub using the Operator Lifecycle Manager
+
+Use this procedure to install KMM-Hub using the Operator Lifecycle Manager (OLM).
```

Comment:
Same thing here. In the Openshift docs we should reference installing KMM-HUB from redhat-operators catalog via UI or via example from M/S docs: 
https://openshift-kmm.netlify.app/documentation/hub_spoke/#on-the-hub

Reply:
Rewrote these sections to refer to RHACM M/S content.